gm4_vecto_shamir/data/gm4_vecto_shamir/predicates/sneaking.json

```
@@ -0,0 +1,14 @@
+{
+  "condition": "minecraft:entity_properties",
+  "entity": "this",
+  "predicate": {
+    "flags": {
+      "is_sneaking": true
+    },
+    "equipment": {
+      "feet": {
+        "nbt": "{gm4_metallurgy:{has_shamir:1b,active_shamir:'vecto'}}"
+      }
```

```suggestion
      "feet": {
        "tag": "gm4:boots",
        "nbt": "{gm4_metallurgy:{has_shamir:1b,active_shamir:'vecto'}}"
      }
```
If you do not check for a specific item or item tag, you will encounter a bug where the predicate incorrectly succeeds despite the boots being unequipped. The solution used for the Celaro shamir was to add a tag to `data/gm4/tags/items/boots.json` and check for it in the predicate. The contents of this tag are as follows:
```json
{
  "values": [
    "minecraft:netherite_boots",
    "minecraft:diamond_boots",
    "minecraft:golden_boots",
    "minecraft:chainmail_boots",
    "minecraft:iron_boots",
    "minecraft:leather_boots"
  ]
}
```

---

This must be why Enderpuffs kept breaking

---

So I need to copy that tag, or should those armor tags be added to the gm4 base, if we need it more than once?

---

I think item tags for each of the armor items is a good idea; #gm4:armor should also be added in case we want all armor items

---

I don't think the tag needs to be in base, but it should be in the `gm4` namespace as the tag is generic and multiple modules may do the same thing, so it makes sense to have a standardized name for a tag that you can copy-paste between modules. This is similar to how every pack includes the `gm4:root` advancement.

---

Does this bug not happen for other shamirs? Or should this be changed in the others too eventually

---

We already have a number of common block tags in the base, I don't see why we shouldn't also make common item tags like for boots and such